### PR TITLE
Feature: assemble individual team page

### LINKED ISSUE #149: Assemble individual team page

### PR DESCRIPTION
**Pull request by rpollock03**
Resolves #149 - assemble individual team page:

Screenshot 2023-02-24 at 13 59 02

Screenshot 2023-02-24 at 13 59 17

### CODE REVIEW

File: components/AboutTeam.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { Typography, Stack, useMediaQuery } from "@mui/material"
+import { Team } from "types/team"
+import { useTheme } from "@mui/material/styles"
+
+interface TeamProps {
```
Comment (piotrostr) [inline components/AboutTeam.tsx:5]: should be `AboutTeamProps`

File: components/Slider/index.tsx [hunk 2]
```diff
@@ -53,10 +59,12 @@ export function Slider({ items, status, variant }: ReusableSliderProps) {
       case SliderVariant.Quest:
         return <LatestQuest quest={item as Quest} />
       case SliderVariant.Role:
-        return <Role role={item as RoleType} teamId={""} />
+        return <Role role={item as RoleType} teamId={variantId} />
     }
   }
 
+  console.log(items)
```
Comment (piotrostr) [inline components/Slider/index.tsx:66]: stranded `console.log`

File: types/team.ts [hunk 3]
```diff
@@ -18,26 +18,26 @@ export enum Industry {
...
+  createdAt?: Date
 }
 
 export interface Role {
-  id: UUID
-  title: string
-  description: string
-  status: string
-  createdAt: Date
+  id?: UUID
```
Comment (piotrostr) [inline types/team.ts:36]: don't make everything optional, `id` and `title` are definitely not optional

File: components/TeamHits/TeamHit.tsx [hunk 4]
```diff
@@ -26,7 +26,7 @@ export function TeamHit({ hit }: LatestTeamProps) {
           <Link
             href={{
               pathname: "/team",
-              query: { teamId: hit.id },
+              query: { tid: hit.id },
```
Comment (piotrostr) [inline components/TeamHits/TeamHit.tsx:29]: why change this?

**Review by piotrostr — CHANGES_REQUESTED** [review]
in the pull requests that are to assemble pages please try not to change any other files as this might backfire

File: components/TeamHits/TeamHit.tsx (hunk 4 above)
Comment (rpollock03) [reply components/TeamHits/TeamHit.tsx:29]: I was making consistent with quest page which uses qid. I can change to questId and back to teamId if you'd prefer.

File: types/team.ts (hunk 3 above)
Comment (rpollock03) [reply types/team.ts:36]: This was a temporary fix, as the incomplete documents/collections on Firestore was causing problems. I have reverted.

**Comment by rpollock03** [comment]
> in the pull requests that are to assemble pages please try not to change any other files as this might backfire

No problem, I do generally try not to but in this case, changes were needed to certain components in order to make this work.

**Pull request merged**